tools/debug/debugsymbolviewer.py

```diff
@@ -140,7 +140,7 @@ def setup_symbol_table(map_file, is_app):
 		print('~~~~~~~~~~~~~~~~~~~~~~~~SYMBOL TABLE END   ~~~~~~~~~~~~~~~~~~~~~')
 
 
-# Function to search the search_addr present in the Address to Symbol mapping table and print its corresponding symbol
+# Function to search the search_addr present in the Address to Symbol mapping table and print its corresponding symbol and related information
```

Comment:
Please change commit subject :
tools/trap: Add below requirements  => tools/trap: Include additional requirements to trap tool.